.buildkite/pipeline.yml

```diff
@@ -172,6 +191,14 @@
     - bazel test --config=ci $(./scripts/bazel_export_options)
```

Comment:
@simon-mo Seems to me those python tests doesn't build C++ code but uses the ray comes with the docker image; 

Does the ray in the docker image reflects the change in the C++ code? (If that's the case, how does this work?/If not, is it a bug or by design?)

Reply:
The docker image is built from scratch in https://github.com/ray-project/ray/blob/master/.buildkite/Dockerfile, which calls https://github.com/ray-project/ray/blob/f1ccadbb2734b7fca54f11e9c9f4fe677fb76b9a/ci/travis/ci.sh#L306-L313

Reply:
ah, thanks!